src/api.jl

```diff
@@ -15,6 +30,7 @@ const TILE_WIDTH = 20
 const DEFAULT_MAX_ITERS = 50
 const MIN_FLUX = 2.0
 
+@inline nputs(nid, s) = ccall(:puts, Cint, (Ptr{Int8},), string("[$nid] ", s))
```

Comment:
This runs "atomically", so the text from multiple nodes isn't interleaved. Also, Logging is used extensively all over Celeste, and I need to turn all that off otherwise there is too much screen spam.